.monorepolint.config.mjs

```diff
@@ -484,13 +485,12 @@ NOTE: DO NOT EDIT THIS README BY HAND. It is generated by monorepolint.
     }),
 
     ...standardPackageRules({
-      includePackages: ["@osdk/foundry-sdk-generator"],
+      includePackages: ["@osdk/e2e.test.foundry-sdk-generator"],
```

Comment:
foundry-sdk-generator is no longer bespoke due to its generated code and e2e tests. But of course the e2e test is still bespoke at least for the generated code. It needs to be excluded because for some reason we are generating an entire package into this space. My goal with this PR was to migrate us to vitest2, not rethink this entire test suite.